### ISSUE TITLE
Why transforms doesnt work (with Vue)?

### ISSUE THREAD

**Issue by hackuun** [julianshapiro/velocity#856]
#### Your system information

* VelocityJS version: ^2.0.2
* Browser: Chromium Version 64.0.3282.167
* Operating System: Ubuntu 16.04

#### Checklist

* Is this an issue with *code*?: [Yes/No] No
* Is this an issue with *documentation*?: [Yes/No] No
* Have you reproduced this in other browsers?: [Yes/No] Yes
* Have you checked for updates?: [Yes/No] Yes
* Have you checked for similar open issues?: [Yes/No] Yes

> Please remember that this is an issue tracker, support requests should be posted on StackOverflow - see CONTRIBUTING.md

#### Please describe your issue in as much detail as possible:

I am trying to use Velocity with Vue.js, following example from Docs - https://vuejs.org/v2/guide/transitions.html#JavaScript-Hooks

All is working (opacity, height), beside all transforms transitions. No translate, rotate etc. Why is this happening?

```js
methods: {
    beforeEnter (el) {
      el.style.opacity = 0
    },
    enter (el, done) {
      Velocity(el, { opacity: 1, fontSize: '1.4em' }, { duration: 300 })
      Velocity(el, { fontSize: '1em' }, { complete: done })
    },
    leave (el, done) {
      Velocity(el, { translateX: '15px', rotateZ: '50deg' }, { duration: 600 })
      Velocity(el, { rotateZ: '100deg' }, { loop: 2 })
      Velocity(el, {
        rotateZ: '45deg',
        translateY: '30px',
        translateX: '30px',
        opacity: 0
      }, { complete: done })
    }
  }
```

**Comment by Rycochet** [comment]
Because `translateX`, `translateY` and `rotateZ` are not valid CSS - use `transform` properties instead.

The Vue.js documentation will need to get updated when Velocity V2 gets out of beta, which should be within a week or two from now.

**Comment by hackuun** [comment]
So I need `transform: "translateX(100px)"`? Where I can find example? Too many options, how do I know how to use it?

**Comment by hackuun** [comment]
Yep, `transform: "translateX(100px)"` is working. Cool, thanks!

**Comment by hackuun** [comment]
@Rycochet I am sorry, but is there anywhere I can track the diff between `1.5.1` and `2.0`?

For example, `'scroll'` is implemented in `2.0` or it's me? Very confusing.

**Comment by Rycochet** [comment]
@hackuun Stated very clearly on the front of this project is that you need to use the wiki and not the website. There's also a V2_CHANGES.md file in the root of the project with a very quick overview of the changes.
